src/apis/posts.js

```diff
@@ -0,0 +1,55 @@
+import { get, del, patch } from "../libs/api/apiWrapper";
+
+import APIs from "../helpers/apiRoutes";
+
+/**
```

Comment:
Doc comments are not up to date

Reply:
Fixed

Reply:
Function doesn't accept params but is in the doc comments

Reply:
Fixed